src/nuxeo-cloud/start-developing.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Start Developing an Application
+review:
+    comment: ''
+    date: '05-14-2020'
+    status: ok
+labels:
+    - nuxeo-cloud
+tree_item_index: 400
+toc: true
+---
+
+## Concept
+
+The Nuxeo Dev Sandbox is built using OpenShift. OpenShift is an open source container application platform, based on top of Docker containers and Kubernetes pods. It enables you to easily build and deploy applications during their development process.
```

Comment:
change this section to 

The Nuxeo Development Sandbox is built on top of Docker containers and Kubernetes pods. Today, we use Openshift to support all Development Sandbox efforts. 

The primary focus of the Development Sandbox is to enable you to easily build and deploy applications during the development process.

The Nuxeo Development Sandbox is meant purely for Development. It does not support large data storage or data backups and should never be used to store production data.